XXX FIXME
GCC complains that nsFrame::delete is protected
..but this really needs to get solved a better way...

diff -up xulrunner-1.9.2.1/mozilla-1.9.2/layout/generic/nsFrame.h.old xulrunner-1.9.2.1/mozilla-1.9.2/layout/generic/nsFrame.h
--- xulrunner-1.9.2.1/mozilla/layout/generic/nsFrame.h.old	2009-11-11 15:34:51.000000000 +0100
+++ xulrunner-1.9.2.1/mozilla/layout/generic/nsFrame.h	2009-11-11 15:33:09.000000000 +0100
@@ -157,7 +157,7 @@ private:
   // Left undefined; nsFrame objects are never allocated from the heap.
   void* operator new(size_t sz) CPP_THROW_NEW;
 
-protected:
+public:
   // Overridden to prevent the global delete from being called, since
   // the memory came out of an arena instead of the heap.
   //
